### PR TITLE
Us 101 2, Adds multi sprint comparison feature(api integration)

### PR DESCRIPTION
**Pull request by AshankDsouza**

### CODE REVIEW

**Review by ShashankReddyBand — DISMISSED** [review]
Do not change the existing burndown component. Please create a new component and either have a option to switch between the charts or have a seperate tab for it

**Review by ShashankReddyBand — CHANGES_REQUESTED** [review]
Change still not addressed

File: client/actions/project.ts [hunk 1]
```diff
@@ -62,18 +62,10 @@ export async function getLeadTime(projectSlug: string, sprintId: string) {
     return leadTimeData;   
 }
 
-export async function getBurndowMetrics(milestoneId: string) {
-    const Response = z.array(z.object({
```
Comment (ShashankReddyBand) [inline client/actions/project.ts:66]: These validations are important we cannot remove them.

File: client/components/burndown.tsx [hunk 2]
```diff
@@ -11,9 +11,8 @@ import {
     SelectValue,
 } from "@/components/ui/select"
 import { useEffect, useState } from "react"
-import BurndownChart from "./burndownChart"
```
Comment (ShashankReddyBand) [inline client/components/burndown.tsx]: Removing this will delete important features

File: client/components/burndown.tsx [hunk 3]
```diff
@@ -23,83 +22,42 @@ function Burndown({slug, sprints}:{slug: string, sprints:{id: string, value: str
 
     const [selectedSprintID, setselectedSprintID] = useState("")
     const [showChart, setShowChart] = useState(true)
-    const [labels, setLabels] = useState<string[]>([])
-    const [open_points, setopen_points] = useState<number[]>([])
-    const [optimal_points, setoptimal_points] = useState<number[]>([])
-    const [story_points, setstory_points] = useState<number[]>([])
```
Comment (ShashankReddyBand) [inline client/components/burndown.tsx]: All this code is important

**Comment by AshankDsouza** [comment]
done with the changes

**Review by ShashankReddyBand — APPROVED** [review]
Looks good

**Pull request merged**